templates/portal/index.html

```diff
@@ -1,11 +1,69 @@
 {% extends "portal/base.html" %}
 {% load allauth i18n %}
 {% load django_bootstrap5 %}
+{% load portal_extras %}
 {% block body %}
 {% endblock body %}
 {% block content %}
     {% if user.is_authenticated %}
-        <div class="container px-4 py-5" id="featured-3">
+        <div class="container my-5">
+            <h2 class="pb-2 border-bottom">Welcome, 
+                <a href="{% url 'volunteer:index' %}">{{ user.first_name }}!</a>
+            </h2>
+            <div class="row mb-4">
```

Comment:
I think I would like to have this section hidden completely if they have not created the Volunteer Profile. Eg the "Volunteer Profile Status" and the "Teams" section should only appear if they have created the profile.

Reply:
@Mariatta , I have added those changes you requested.

When they have not created the Volunteer Profile:

<img width="1505" alt="Screenshot 2025-05-19 at 5 26 40 PM" src="https://github.com/user-attachments/assets/78984fa5-3dde-4d46-a93f-cb324f85a036" />

once it has been created:
<img width="1497" alt="Screenshot 2025-05-19 at 5 27 13 PM" src="https://github.com/user-attachments/assets/d9c1b82d-ca49-4d4e-97bb-edb4417140ac" />